### PR TITLE
Clean up tables on load

### LINKED ISSUE #787: Clean up demo database upon restart
The application should clean up the demo database (i.e. delete `institutions`, `lar` tables if present) when starting. If this isn't done automatically, it's necessary to run `clean` from the `api` project before running it

### PR DESCRIPTION
**Pull request by nickgrippin**
Closes #787

### CODE REVIEW

File: api/src/main/scala/hmda/api/HmdaPlatform.scala [hunk 1]
```diff
@@ -39,8 +43,25 @@ object HmdaPlatform {
 
   }
 
+  private def cleanup(system: ActorSystem, supervisor: ActorRef, querySupervisor: ActorRef)(implicit ec: ExecutionContext): Unit = {
```
Comment (jmarin) [inline api/src/main/scala/hmda/api/HmdaPlatform.scala]: Do you still need `supervisor` / `querySupervisor` with this approach?

**Comment by jmarin** [comment]
Just a question about the `cleanup` function signature, otherwise LGTM

**Comment by schbetsy** [comment]
I tried running the Panel CSV Parser and got this error:
```
[error] (run-main-0) org.postgresql.util.PSQLException: ERROR: relation "institutions" already exists
org.postgresql.util.PSQLException: ERROR: relation "institutions" already exists
```

Would it be out of scope for this PR to solve that failure too?

**Comment by schbetsy** [comment]
How does this PR interact with the Panel CSV Parser PR? I tried loading panel data from a file then starting the project (like in the instructions in #797), but I just saw Demo data instead.

**Comment by nickgrippin** [comment]
@schbetsy: make sure you have the `HMDA_IS_DEMO` environment variable set to `false`.  Let me know if that works for you.

**Comment by schbetsy** [comment]
The environment variable did the trick. Thank you.

I think the only remaining question is [this one](https://github.com/cfpb/hmda-platform/pull/801#issuecomment-278774276), about the PSQL Exception in the Panel CSV Parser.

**Pull request merged**